tests/tasks.py

```diff
@@ -293,3 +292,40 @@ def mytask(longer_arg):
             eq_(arg.names, ('longer-arg', 'l'))
             eq_(arg.attr_name, 'longer_arg')
             eq_(arg.name, 'longer_arg')
+
+        def decorated_function_names_are_preserved(self):
```

Comment:
That setup lets you load individual test files with `-m` (so e.g. `inv test -m runners` to run `tests/runners.py`). spec lacks support for individual tests, which is annoying and something I'll probably try to get working soon.